api/services/ocean/cd/paths/rollout.yaml

```diff
@@ -1,4 +1,25 @@
 summary: Ocean CD
+get:
+  summary: "Ocean CD Rollout by ID"
+  description: >
+    Ocean CD single rollout.
+  operationId: "OceanCDRolloutSingle"
```

Comment:
@lionelresnik - Lets change this to DescribeRollout (And every other get)

Reply:
changed